source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+
+```
+{
+  "next": {
+    "id": "https://example.org/activity/page-2",
+    "type": "OrderedCollectionPage"
+  }
+}
+```
+
+##### prev
+
+A reference to the previous page in the list of pages.
+
+Ordered Collection Pages _MUST_ have a `prev` property, unless they are the first page in the Collection. The value _MUST_ be a JSON object, with the `id` and `type` properties.  The value of the `id` property _MUST_ be the a string, and _MUST_ be the HTTP(S) URI of the preceding Ordered Collection Page.  The value of the `type` property _MUST_ be a string, and _MUST_ be `OrderedCollectionPage`.
+
+```
+{
+  "prev": {
+    "id": "https://example.org/activity/page-1",
+    "type": "OrderedCollectionPage"
+  }
+}
+```
+
+##### orderedItems
+
+The Activities that are listed as part of this page.
+
+Ordered Collection Pages _MUST_ have a `orderedItems` property.  The value _MUST_ be an array, with at least one item.  Each item _MUST_ be a JSON object, conforming to the requirements of an Activity.
+
+```
+{
+  "orderedItems": [
+     {
+     	"type": "Update",
+     	"object": {
+     		"id": "https://example.org/iiif/1/manifest",
+     		"type": "Manifest"
+     	},
+     	"endTime": "2018-03-10T10:00:00Z"
+     }
+  ]
+}
+```
+
+
+##### Complete Ordered Collection Page Example
+
+```
+{
+  "@context": "http://iiif.io/api/discovery/0/context.json",
+  "id": "https://example.org/activity/page-1",
+  "type": "OrderedCollectionPage",
+  "startIndex": 20,
+  "partOf": {
+    "id": "https://example.org/activity/all-changes",
+    "type": "OrderedCollection"
+  },
+  "prev": {
+    "id": "https://example.org/activity/page-0",
+    "type": "OrderedCollectionPage"
+  },
+  "next": {
+    "id": "https://example.org/activity/page-2",
+    "type": "OrderedCollectionPage"
+  },
+  "orderedItems": [
+    {
+      "type": "Update",
+      "object": {
+        "id": "https://example.org/iiif/1/manifest",
+        "type": "Manifest"
+      },
+      "endTime": "2018-03-10T10:00:00Z"
+    }
+  ]
+}
+```
+
+### 3.3. Activities
+{: #activities}
+
+The Activities are the means of describing the changes that have occured in the content provider's system.
+
+Content providers _MAY_ publish Activities separately from Ordered Collection Pages, and if so they _MUST_ be at the HTTP(S) URI given in the `id` property of the Activity.
+
+##### id
+
+An identifier for the Activity.
+
+Activities _MAY_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Activity _MAY_ be available at the URI.
+
+```
+{ "id": "https://example.org/activity/1" }
+```
+
+##### type
+
+The type of Activity. 
+
+This specification uses the types described in the table below.
+
+| Type   | Definition |
+| ------ | ---------- |
+| `Create` | The initial creation of the resource.  Each resource _MUST_ have at most one `Create` Activity in which it is the `object`. |
```

Comment:
Is it intentionally illegal to have a `Create` -> `Update` -> `Delete` -> `Create` -> `Update` sequence for a resource? (per max one `Create`, max one `Delete` two lines below). Maybe this is not an editorial question for now but I wonder whether this is a useful/necessary restriction on AS

Reply:
Will make an issue for this, as I think it's normative rather than just editorial.

Reply:
Done, https://github.com/IIIF/discovery/issues/50